.changeset/tender-mangos-wash.md

```diff
@@ -0,0 +1,7 @@
+---
+"@sap/guided-answers-extension-core": patch
+"sap-guided-answers-extension": patch
+"@sap/guided-answers-extension-webapp": patch
+---
+
+fix(ga): issue #606 - search inconsistent
```

Comment:
Can we change it to something like

Fixed issues with the search
- add debounce to get consistent search results
- fixed an issue where the loading indicator was shown even when there were no more search results to fetch

Reply:
Done